### PR TITLE
Adding Action Announcement & prompt - Fixing #8107

### PR DESCRIPTION
**Pull request by DEberhardt**
PR to add the Action Announcement and the Parameter Prompt to New-CsAutoAttendantMenuOption

### CODE REVIEW

File: skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md [hunk 1]
```diff
@@ -43,12 +43,21 @@ $menuOption = New-CsAutoAttendantMenuOption -Action TransferCallToTarget -DtmfRe
 
 This example creates a menu option to transfer the call to an application endpoint when the caller speaks the word "Sales" or presses the 1 key.
 
+### -------------------------- Example 3 --------------------------
+```
+$Prompt = New-CsAutoAttendantPrompt -ActiveType -TextToSpeechPrompt "Our Office is open from Monday to Friday from 9 AM to 5 PM"
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md:48]: Please check the cmdlet as I received an error:
`New-CsAutoAttendantPrompt : Missing an argument for parameter 'ActiveType'. Specify a parameter of type
'System.Object' and try again.`

**Review by get-itips — COMMENTED** [review]
Please also add `-Prompt` to the parameter set section

File: skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md (hunk 1 above)
Comment (dariomws) [reply skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md:48]:
```suggestion
$Prompt = New-CsAutoAttendantPrompt -ActiveType TextToSpeech -TextToSpeechPrompt "Our Office is open from Monday to Friday from 9 AM to 5 PM"
```
Agree with @get-itips, I think the value for ActiveType should be TextToSpeech.

**Review by dariomws — COMMENTED** [review]
Agree with @get-itips, -Prompt parameter must be added to syntax section.

**Comment by DEberhardt** [comment]
I am too coddled by PlatyPS... always forget to update the Syntax itself.^^
Done.

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws and @tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Comment by yogkumgit** [comment]
@dariomws

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md [hunk 2]
```diff
@@ -43,12 +43,21 @@ $menuOption = New-CsAutoAttendantMenuOption -Action TransferCallToTarget -DtmfRe
 
 This example creates a menu option to transfer the call to an application endpoint when the caller speaks the word "Sales" or presses the 1 key.
 
+### -------------------------- Example 3 --------------------------
+```
+$Prompt = New-CsAutoAttendantPrompt -ActiveType TextToSpeech -TextToSpeechPrompt "Our Office is open from Monday to Friday from 9 AM to 5 PM"
+$menuOption = New-CsAutoAttendantMenuOption -Action Announcement -DtmfResponse Tone2 -VoiceResponses "Hours" -Prompt $Prompt
+```
+
+This example creates a menu option to play an Announcement for the defined Prompt. After playing the Announcement, the Menu Prompt is repeated.
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md:52]:
```suggestion
This example creates a menu option to play an announcement for the defined prompt. After playing the announcement, the **Menu Prompt** is repeated.
```

File: skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md [hunk 3]
```diff
@@ -126,6 +135,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Prompt
+The Prompt parameter reflects the prompts to play when the menu option is activated.
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md:139]:
```suggestion
The **Prompt** parameter reflects the prompts to play when the menu option is activated.
```

File: skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md [hunk 4]
```diff
@@ -126,6 +135,26 @@ Accept pipeline input: False
...
+
+You can create new prompts by using the New-CsAutoAttendantPrompt cmdlet.
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md:141]:
```suggestion
You can create new prompts by using the **New-CsAutoAttendantPrompt** cmdlet.
```

File: skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md [hunk 5]
```diff
@@ -126,6 +135,26 @@ Accept pipeline input: False
...
+
+This parameter is required if the Action has been set to Announcement.
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md:143]:
```suggestion
This parameter is required if the **Action** is set to **Announcement**.
```

**Pull request merged**